public/tools/rmt-client-setup

```diff
@@ -128,6 +126,9 @@ function import_signing_keys()
                 fi
             fi
         done
+    elif [ $retcode -ne 22 ]; then
+      local key_file_url=`echo "$REGURL/$1" | awk -F/ '{print "https://" $3 "/repo/keys/" $4}'`
+      echo "An error occurred reading the custom rpm key directory $key_file_url. Please ensure this directory exists, is readable to RMT and contains rpm signing keys if you wish to import custom rpm signing keys."
```

Comment:
I wonder if we need this message at all -- I'm not sure under what circumstances this code could be ever executed.
The message at least shouldn't imply that any action is required, because the user might not have any keys at all.